### PR TITLE
Adds disabled preview modal when blank content on Quill Editor

### PR DESCRIPTION
**Pull request by martins87**
## Link to Issue
Closes: #4825 

## Description of Changes
- Disabled preview when blank content

## "How We Fixed It"

Simple check for string length returned by `getTextFromDelta` function.

## Test Plan
- Tested with text
- Tested with images

## Other considerations

The `getTextFromDelta` will return `''` on blank text or content (no images or twitter). Will return length > 0 when some content present:
- `'(tweet)'` for twitter content
- `'(image)\n'` for image content

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx [hunk 1]
```diff
@@ -234,6 +234,7 @@ const ReactQuillEditor = ({
               handleToggleMarkdown={handleToggleMarkdown}
               setIsPreviewVisible={setIsPreviewVisible}
               isDisabled={isDisabled}
+              content={getTextFromDelta(contentDelta)}
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx:237]: you can pass just a bool down like so

`isPreviewDisabled={content?.length < 1}`

Then in `toolbar.tsx` you will add only one line 
`disabled={isPreviewDisabled}`

Comment (martins87) [reply packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx]: Addressed.

**Review by masvelio — APPROVED** [review]
approved the code, I did not QA on app

**Review by jnaviask — APPROVED** [review]

**Comment by sachben91** [comment]
- [ ] tested quill editor and preview is disabled when content is blank

**Pull request merged**